cfme/tests/configure/test_tag_category.py

```diff
@@ -39,8 +39,8 @@ def test_create_categories(self, appliance, categories):
         """
         for ctg in categories:
             record = appliance.rest_api.collections.categories.get(id=ctg.id)
-            assert appliance.rest_api.response.status_code == 200
-            assert record.name == ctg.name
+        assert_response(appliance)
+        assert record.name == ctg.name
```

Comment:
REQUIRED: indentation is wrong, this should be inside of the loop

Reply:
fixed